addons/resource.language.en_gb/resources/strings.po

```diff
@@ -1036,7 +1040,11 @@ msgctxt "#222"
 msgid "Cancel"
 msgstr ""
 
-#empty string with id 223
+#: xbmc/games/dialogs/GUIDialogSelectSavestate.cpp
+#: xbmc/games/dialogs/osd/DialogGameSaves.cpp
+msgctxt "#223"
+msgid "New"
```

Comment:
I just tested `xbmc/games/dialogs/GUIDialogSelectSavestate.cpp` and think that `New` would be better as `New game` (ok for the other dialog though)
![New game](https://user-images.githubusercontent.com/531482/87894984-20a5d500-c9f9-11ea-9ac4-92571eff7ac2.png)